.github/workflows/ci.yml

```
@@ -31,7 +31,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2.1.5
```

This will create less PRs

```suggestion
        uses: actions/setup-node@v2
```